lib/nodes.js

```diff
@@ -1,15 +1,33 @@
 import * as address from "./nodes/address.js";
+import * as blockquote from "./nodes/blockquote.js";
+import * as bullet_list from "./nodes/bullet_list.js";
 import * as callToAction from "./nodes/call_to_action.js";
 import * as contact from "./nodes/contact.js";
+import * as doc from "./nodes/doc.js";
 import * as exampleCallout from "./nodes/example_callout.js";
+import * as hard_break from "./nodes/hard_break.js";
+import * as heading from "./nodes/heading.js";
 import * as informationCallout from "./nodes/information_callout.js";
+import * as list_item from "./nodes/list_item.js";
+import * as ordered_list from "./nodes/ordered_list.js";
+import * as paragraph from "./nodes/paragraph.js";
+import * as text from "./nodes/text.js";
 import * as warningCallout from "./nodes/warning_callout.js";
 
 export default [
+  paragraph, // Paragraph must be first to be the default
   address,
+  blockquote,
+  bullet_list,
   callToAction,
   contact,
+  doc,
   exampleCallout,
+  hard_break,
+  heading,
   informationCallout,
+  list_item,
+  ordered_list,
+  text,
```

Comment:
Might be a nice touch at some point to export different node lists from this file, e.g. `nodesWithMenuItems`, `govspeakNodes` etc. to save ourselves checking various node properties aren't undefined before we call them.

Reply:
Good point it's quite clunky as is, I'll have a think about how to approach it.
It might also be nice to just centralise the mapping from node definitions to whats needed in `nodes.js`